contracts/strategy/src/contract.rs

```diff
@@ -161,6 +161,7 @@ fn prepare_delegation_data(
         let delegation = drop_staking_base::msg::distribution::Delegation {
             valoper_address: validator.valoper_address.clone(),
             stake: validator_denom_delegation,
+            on_top: Uint128::zero(),
```

Comment:
do I get it right that it's just a placeholder now?

Reply:
yes, it's just a placeholder for future, this PR only targets distribution contract